### DISCUSSION TITLE
Update 1.3.8

### DISCUSSION THREAD

**Discussion by JsonBE** [TobiasM95/WoW-Talent-Tree-Manager#15] [General]
@TobiasM95 Any idea when 1.3.8 would be released with what currently is available for DH?

**Comment by TobiasM95** [comment]
Version releases are decoupled from preset updates. One DH spec is already in TTM with its respective icons. I would have to check if the other got released mid week. Then I can push that as well.
As for 1.3.8, since there aren't any major performance/stability issues right now it will not be released that soon without any major features.

**Reply by JsonBE** [reply]
I saw that the update was already in the .sln but not available as .exe yet, is that correct?

**Reply by TobiasM95** [reply]
Parts of 1.3.8 are in the master branch and therefore available to pull if you cloned the repository. Obviously that doesn't represent the full 1.3.8 update

**Reply by JsonBE** [reply]
@TobiasM95 the other one got released as well, when can we expect the push coming through?
Kind regards,
JsonBE

**Reply by TobiasM95** [reply]
I wanted to wait until the weekly patch to push new presets. Then Friday got busy with 1.3.8 stuff and new Feral changes.
Everything is up to date now with Feral having the preliminary tree variant.

**Reply by JsonBE** [reply]
Fracture Icon for VDH is the wrong Icon

**Reply by TobiasM95** [reply]
errors like this can occur from time to time. will be fixed down the line but is not high priority
